submissions/jeremy-sung/index.html

```diff
@@ -33,9 +33,9 @@
         <section id="articles">
           <article class="template" data-category="cat">
             <header>
-              <h1>Title</h1>
+              <h1 id="article-title">Title</h1>
               <div class="byline">
-                By <address><a href="">Author Name</a></address>
+                By <address id="article-author"><a id="article-authorUrl" href="">Author Name</a></address>
                 published <time pubdate datetime="2000-01-01">Publish Time</time>
```

Comment:
I see now what you were saying, Jeremy. No worries! Specificity is not as important to us in this case as our targeted elements can be accessed with jQuery's .find() method on the $newArticle